### ISSUE TITLE
Upgrading to 11.12.0 fails with esbuild/install.js Error: Expected "0.15.8" but got "0.14.51"

### ISSUE THREAD

**Issue by davidmytton** [firebase/firebase-tools#5049]
### [REQUIRED] Environment info

**firebase-tools:** Upgrading from 11.11.0 -> 11.12.0

**Platform:** macOS 12.6 (21G115) `Darwin dm-mba-m1.local 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:20:05 PDT 2022; root:xnu-8020.140.49~2/RELEASE_ARM64_T8101 arm64`

node `v18.9.0`

### [REQUIRED] Steps to reproduce

`firebase-tools` 11.11.0 is installed without any issues in `devDependencies`. Changing the `package.json` version to `11.12.0` and running `npm install` fails.

### [REQUIRED] Expected behavior

The install/upgrade should succeed.

### [REQUIRED] Actual behavior

```
npm ERR! code 1
npm ERR! path /Users/david/Code/jobs-console/web/node_modules/firebase-tools/node_modules/esbuild
npm ERR! command failed
npm ERR! command sh -c -- node install.js
npm ERR! /Users/david/Code/jobs-console/web/node_modules/firebase-tools/node_modules/esbuild/install.js:94
npm ERR!     throw new Error(`Expected ${JSON.stringify("0.15.8")} but got ${JSON.stringify(stdout)}`);
npm ERR!           ^
npm ERR! 
npm ERR! Error: Expected "0.15.8" but got "0.14.51"
npm ERR!     at validateBinaryVersion (/Users/david/Code/jobs-console/web/node_modules/firebase-tools/node_modules/esbuild/install.js:94:11)
npm ERR!     at /Users/david/Code/jobs-console/web/node_modules/firebase-tools/node_modules/esbuild/install.js:243:5
npm ERR! 
npm ERR! Node.js v18.9.0
```

Log attached
[log.log](https://github.com/firebase/firebase-tools/files/9683923/log.log)

**Comment by bkendall** [comment]
@davidmytton I'm gonna _guess_ that this should be resolved by #5052. I'm going to be pushing that release button very soon, so keep an eye out for a new minor version (11.13.0) and see if it's still an issue, please.

**Comment by calvinturbo** [comment]
@bkendall I can confirm the issue has been resolved

**Comment by bkendall** [comment]
👍  thanks for confirming!

**Comment by davidmytton** [comment]
Also confirmed fixed in 11.13.0. Thanks!
